Apps/Playground/UWP/App.cpp

```diff
@@ -138,16 +145,7 @@ void App::OnActivated(CoreApplicationView^ applicationView, IActivatedEventArgs^
 
 concurrency::task<void> App::RestartRuntimeAsync(Windows::Foundation::Rect bounds)
 {
-    if (m_inputBuffer)
-    {
-        m_inputBuffer.reset();
-    }
-
-    if (m_runtime)
-    {
-        m_runtime.reset();
-        Babylon::DeinitializeGraphics();
-    }
+    Uninitialize();
```

Comment:
There is some code below that isn't necessary anymore?

```
    // Ensure this is properly uninitialized since it depends on state of the runtime.
    m_inputBuffer.reset();
```